global.json

```diff
@@ -3,13 +3,13 @@
     "allowPrerelease": true
   },
   "tools": {
-    "dotnet": "8.0.100-rc.1.23463.5",
+    "dotnet": "8.0.100-rtm.23506.1",
```

Comment:
Not sure we can do this yet, I think it'll break our official builds because they're not on VS 17.8 yet.

Reply:
Should we revert 8.0.100-rtm.23506.1 to 8.0.100-rc.1.23463.5?

Reply:
Yes, that should be fine.

Reply:
Reverted